src/organization/screens/organization-profile.screen.js

```diff
@@ -143,6 +143,7 @@ class OrganizationProfile extends Component {
             <MembersList
               title={t('MEMBERS', locale)}
               members={orgMembers}
+              noMembersMessage={t('No contributors found', locale)}
```

Comment:
For an org, this seems a little better fit:

```suggestion
              noMembersMessage={t('No members found', locale)}
```